dsc_lib/src/configure/mod.rs

```diff
@@ -135,13 +135,13 @@ fn escape_property_values(properties: &Map<String, Value>) -> Result<Option<Map<
     Ok(Some(result))
 }
 
-fn get_progress_bar(len: u64) -> Result<ProgressBar, DscError> {
-    let pb = ProgressBar::new(len);
-    pb.enable_steady_tick(Duration::from_millis(120));
-    pb.set_style(ProgressStyle::with_template(
+fn get_progress_bar_span(len: u64) -> Result<Span, DscError> {
+    let pb_span = warn_span!("progress bar");
```

Comment:
If we just have this be an empty string `warn_span!("")` would that suppress the unnecessary `progress bar` in the traces?